common/test/acceptance/tests/lms/test_account_settings.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+"""
+End-to-end tests for the Student Account Settings.
+"""
+from ...pages.lms.auto_auth import AutoAuthPage
+from ...pages.lms.account_settings import AccountSettingsPage
+from ...pages.lms.dashboard import DashboardPage
+
+from bok_choy.web_app_test import WebAppTest
+
+
+class AccountSettingsTest(WebAppTest):
+    """
+    Tests that verify Student Account Settings Page.
+    """
+
+    USERNAME = 'test_user'
+    EMAIL = 'test_user@edx.org'
+
+    def setUp(self):
+        """
+        Initialize pages and install a course fixture.
+        """
+        super(AccountSettingsTest, self).setUp()
+
+        self.account_settings_page = AccountSettingsPage(self.browser)
+        self.dashboard_page = DashboardPage(self.browser)
+
+        AutoAuthPage(self.browser, username=self.USERNAME, email=self.EMAIL).visit()
+
+    def test_dashboard_account_settings_link(self):
+        """
+        Scenario: Verify that account settings link is present on dashboard page and we can navigate to it.
+
+        Given that I am a registered user.
```

Comment:
It turns out that this capability works for both verified users as well as for users who are still in the registration workflow. It might be worth having a test for each, because my understanding is that using settings during registration is important so that the user can specify their desired language for edX.